interface_tester/collector.py

```diff
@@ -152,19 +153,19 @@ def get_schemas(file: Path) -> Dict[Literal["requirer", "provider"], Type[DataBa
 
 
 def _gather_charms_for_version(version_dir: Path) -> Optional[_CharmsDotYamlSpec]:
-    """Attempt to read the `charms.yaml` for this version sudir.
+    """Attempt to read the `interface.yaml` for this version sudir.
```

Comment:
Could "sudir" be a typo? Maybe it's "subdir" or "directory"? Probably not "su[do] dir"?

Reply:
Or "version's dir"? Or `version_dir` to match the arg.

Reply:
also, please rename the `_CharmsDotYamlSpec` to ``_InterfacesDotYamlSpec` or something more telling